### PR TITLE
Rule number one

### PR DESCRIPTION
**Pull request by Staslgvin23**
Here is the pull request for my review. Notice: you can comment any line in this PR changelog and I'll be notified. I will answer as soon as possible.

### CODE REVIEW

**Comment by Staslgvin23** [comment]
When we finish discussing this changes we will merge them to your master branch

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-  <h1>Parallax effect</h1>
-  <p>Pure CSS. No JS</p>
-  <span>
-    <p>This is my first working session in VS Code.</p>
-    <p>Also I Use GitBusher for virtual server creation,<br>
-    and GitKraken for version control and changes commitment.</p>
-  </span>
-</header>
-<div class="logo">
+<div class="body-block">
```
Comment (niknorp) [inline src/app/app.component.html]: Did You add .body-block as a wrapper for all page content?

File: src/app/app.component.css [hunk 2]
```diff
@@ -1,49 +1,16 @@
...
-  right: 0;
-  top:0;
-  display: block;
-  background-image: url(../img/me.jpg);
-  background-size: cover;
-  transform-origin: center center 0;
-  transform: translateZ(-1px) scale(2);
-  z-index: -1;
-  min-height: 100vh;
-}
```
Comment (niknorp) [inline src/app/app.component.css]: Sorry,
But I don't understand why are you remove these lines? (32-46)

**Comment by niknorp** [comment]
I think that I understood changes wich are made by you.
You moved content from index.html and styles.css to appropriate files, removed global styling, add .body-block as a page wrapper (if I understand well) and moved background-img to appropriate place.
But I don't understand delete of "header::before" lines.

**Review by niknorp — CHANGES_REQUESTED** [review]
I think that I gennerally understood changes wich are made by you.
You moved content from index.html and styles.css to appropriate files, removed global styling, add .body-block as a page wrapper (if I understand well) and moved background-img to appropriate place.
But I don't understand delete of "header::before" lines.

File: src/app/app.component.html (hunk 1 above)
Comment (Staslgvin23) [reply src/app/app.component.html]: Yes, just to make sure that it will be rendered as a one tag at first

**Review by Staslgvin23 — COMMENTED** [review]
I deleted some styles only because they were too general (for `body` and `html` tags), or the measures that you used (avoid `vh` and `vw` when it's possible)

**Pull request merged**